src/main/java/com/ombremoon/enderring/common/init/entity/StatusEffectInit.java

```diff
@@ -136,7 +138,7 @@ public class StatusEffectInit {
                     .addAttribute(new AttributeModifier(
                                     UUID.fromString("d8b7d3c-04cc-64b3-6b08-67b8bdb40834"),
                                     "SPECKLEDHARD_CRYSTAL",
-                                    6F, AttributeModifier.Operation.ADDITION),
+                                    90F, AttributeModifier.Operation.ADDITION),
```

Comment:
Should increases all resistances by 90. That includes Robustness, Focus, and Vitality